payments/views.py

```diff
@@ -180,7 +178,8 @@ def increase_balance(event_body, topup):
 @method_decorator(login_required, name='dispatch')
 class PaymentHistoryView(View):
     def get(self, request):
-        user_payments = TopUp.payments.filter(user=request.user).filter(~Q(payment_intent_status='')).order_by('-date_created')
+        user_payments = TopUp.payments.filter(user=request.user).filter(~Q(payment_intent_status='')).order_by(
```

Comment:
`TopUp.payments.filter(user=request.user).exclude(payment_intent_status='').order_.....`

Reply:
Depreciated; modified to ListView